### ISSUE TITLE
App status

### ISSUE THREAD

**Issue by vasnt** [ScientiaLiber/MindIndex#51]
- is this app open for public?
- or app development is still going on?
- How it compares to anki/remnote?

- which features available?
- importing text/csv available?

**Comment by Md7tz** [comment]
@vasnt Yes, the app is publicly available for use. However, please note that it is still under development and some features may be incomplete or subject to further updates., the main features are almost done.

Features and How it compares to Anki?

CRUD collections/study-sets (grouping flashcards):
With this app, you can create, organize, and manage your flashcards into different collections or study sets. This helps you keep your flashcards grouped based on different subjects or topics you're studying.

CRUD flashcards:
You have the flexibility to create, edit, and delete individual flashcards. This means you can easily add new information, make changes to existing flashcards, or remove any flashcards that are no longer needed.

CRUD notes, linking notes to study sets:
In addition to flashcards, you can also create and manage notes within the app. These notes can be linked to your study sets, allowing you to supplement your flashcards with additional information or explanations.

Full-text search:
Finding specific flashcards or notes is a breeze with the full-text search feature. Simply enter a keyword or phrase, and the app will quickly search and display relevant results, helping you locate the information you need.

Payment gateway subscription:
The app provides a subscription option through a secure payment gateway. This means you can access additional premium features or services by subscribing, enhancing your studying experience with extra benefits (removing limits on the number of flashcards, study-sets, notes which is reset every month).

Authentication:
To ensure your data is secure and personalized, the app includes user authentication. This allows you to create your own account, log in with your credentials, and protect your study materials with a password.

User profile:
Within the app, you have a user profile where you can customize your settings and preferences. This includes personalizing your profile information, adjusting notification preferences, and tailoring the app to suit your study habits.

Please note that data import by CSV or SQL is not yet supported in the upcoming v1.0, which is scheduled to be released this week.

**Comment by vasnt** [comment]
followup queries:
- Any video demo available?
- Any AI feature supported/planned? e.g. Automatic flashcards generation
